src/monorepo_manifest.rs

```diff
@@ -47,34 +51,37 @@ where
     // ignore paths to speed up file-system walk
     package_manifests.push(String::from("!node_modules/"));
 
+    let monorepo_root = directory.as_ref().to_owned();
+
     GlobWalkerBuilder::from_patterns(&directory, &package_manifests)
         .file_type(FileType::FILE)
         .min_depth(1)
         .build()
         .expect("Unable to create glob")
         .into_iter()
         .filter_map(Result::ok)
```

Comment:
Ignoring errors - not a best strategy. For simplicity, I'd just `.map(|r| r.unwrap())` to cause a panic on any error.

Reply:
Properly handling it is a bit of a PITA, if you don't want to `collect` to `Result<Vec<_>>` as an intermediate step (which you don't here). Though it is possibly. you just have to handle&pass `Result` everywhere. :shrug:

Reply:
This is a fair point. I'm going to punt on this for now as it would be a breaking change